release-notes.md

```diff
@@ -0,0 +1,51 @@
+# v0.1.0
+
+This release has been tracked through the [0.1.0 project](https://github.com/intel/cloud-hypervisor/projects/1).
+
+Highlights for `cloud-hypervisor` version 0.1.0 include:
+
+### Shared filesystem
```

Comment:
Should mention virtio-pmem!

Reply:
Should mention we now support loading a bzImage.

Reply:
Right, I was only looking at the 0.1.0 project cards...
I added a section for both features.